usr/share/lipstick-jolla-home-qt5/launcher/LauncherView.qml

```diff
@@ -4,6 +4,9 @@ import org.nemomobile.lipstick 0.1
 import Sailfish.Silica 1.0
 import Sailfish.Silica.private 1.0
```

Comment:
[Line 5 of usr/share/lipstick-jolla-home-qt5/launcher/LauncherView.qml](https://github.com/sailfishos-patches/sailfish-qml/pull/1/files#diff-895c7ca4bb1d77d0da36f0940246aa495e2bd7e4850d95694ba4d643ed532f7dR5) became superfluous by [adding line 9](https://github.com/sailfishos-patches/sailfish-qml/pull/1/files#diff-895c7ca4bb1d77d0da36f0940246aa495e2bd7e4850d95694ba4d643ed532f7dR5) and utilising [solely `SilicaPrivate` in `usr/share/lipstick-jolla-home-qt5/launcher/LauncherView.qml`(https://github.com/sailfishos-patches/sailfish-qml/pull/1/files#diff-895c7ca4bb1d77d0da36f0940246aa495e2bd7e4850d95694ba4d643ed532f7dR37).